src/style.css

```diff
@@ -0,0 +1,123 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.product__card {
+  color: #060b35;
+
+  margin: 10px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  display: flex;
+  justify-content: center;
+
+  margin-top: 32px;
+  margin-bottom: 40px;
```

Comment:
Need to use only top or only bottom margins, like in CHECKLIST:
'[STYLES] - Be consistent with your margins. Add only top or bottom, don't add both.'